website/frontend/static/css/styles.css

```diff
@@ -5520,3 +5520,11 @@ pre {
   white-space: nowrap;
   border-radius: 4px;
 }
+
+.ddm{
+  display: none;  
+}
+
+.download-btn:hover .ddm{
+  display: block;
```

Comment:
I didn't wanted to change the very super class as if the same classes would
have been used somewhere else where we dont want this effect. Will do the
same if suggested.

On Wed, Feb 18, 2015 at 4:43 AM, Philipp Wolfer <notifications@github.com>
wrote:

> In website/frontend/static/css/styles.css
> <https://github.com/musicbrainz/picard-website/pull/24#discussion_r24864659>
> :
>
> > @@ -5520,3 +5520,11 @@ pre {
> >    white-space: nowrap;
> >    border-radius: 4px;
> >  }
> > +
> > +.ddm{
> > +  display: none;
> > +}
> > +
> > +.download-btn:hover .ddm{
> > +  display: block;
>
> Why the extra ddm class (stands for dropdown menu?), .download-btn:hover
> .dropdown-menu would be just fine and nicer to read. It also looks weird
> in the HTML (this element is a dropdown-menu and also a ddm!?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/musicbrainz/picard-website/pull/24/files#r24864659>.
>